src/js/panels.js

```diff
@@ -106,8 +106,9 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
     };
 
     gpii.firstDiscovery.panel.ranged.updateButtonState = function (that) {
-        var isMax = that.model.value >= that.options.range.max;
-        var isMin = that.model.value <= that.options.range.min;
+        // assumes that values can't go out of range (based on range limit modelRelay)
+        var isMax = fluid.model.isSameValue(that.model.value, that.options.range.max);
+        var isMin = fluid.model.isSameValue(that.model.value, that.options.range.min);
```

Comment:
Due to (FLUID-5669)[http://issues.fluidproject.org/browse/FLUID-5669] I had to use fluid.transforms.free with a call to `fluid.model.isSameValue`, rather than `fluid.transforms.binaryOp`. Also because the `updateButtonState` method is to update DOM elements, it needs to be fired afterRender.